tests/interface/TWHarmonySignerTests.cpp

```diff
@@ -0,0 +1,44 @@
+
```

Comment:
This is just a copy of `TWEthereumSignerTests`?  Can you add a `TWHarmonySignerSign` test case? and remove duplicated `EmptyValue` and `BigInt` tests

Reply:
Ah sorry, I overlooked this one.